apps/src/applab/commands.js

```diff
@@ -1381,7 +1384,9 @@ applabCommands.startWebRequest = function (opts) {
   apiValidateType(opts, 'startWebRequest', 'callback', opts.func, 'function');
   var req = new XMLHttpRequest();
   req.onreadystatechange = applabCommands.onHttpRequestEvent.bind(req, opts);
-  req.open('GET', opts.url, true);
+  var url = XHR_PROXY_PATH + '?u=' + encodeURIComponent(opts.url.replace(/ /g, '%20')) +
+      '&c=' + Applab.channelId;
```

Comment:
We should probably encode the channel id as well, just in case it ever evolves to include reserved characters.  Is there any possibility of it being null where we would want to return an error?